ptd_client_server/lib/server.py

```diff
@@ -268,6 +279,7 @@ def get(
             parse=get_host_port_from_listen_string,
             fallback=f"0.0.0.0 {common.DEFAULT_PORT}",
         )
+        self.range_mode = get("server", "rangeMode", fallback="AUTO")
```

Comment:
I'd use "ranging mode" instead of "range mode" here and elsewhere, as we call the first run "ranging", not "range".